modules/config_games/server_configs/wurmu_win64.xml

```diff
@@ -0,0 +1,94 @@
+<game_config>
+  <game_key>wurmu_win64</game_key>
+  <installer>steamcmd</installer>
+  <game_name>Wurm Unlimited Server Windows</game_name>
+  <server_exec_name>WurmServerLauncher</server_exec_name>
+  <cli_template>%IP% %PORT% %QUERY_PORT% %PLAYERS% %SERVERNAME% %SERVERPASS% %ADMINPASS% %EPIC% %HOME% %HOMEKING% %LOGIN% %PVP% %RMI% %RMIREG% %START%</cli_template>
+  <cli_params>
+    <cli_param id="IP" cli_string="ip=" />
+    <cli_param id="PORT" cli_string="externalport=" />
+    <cli_param id="QUERY_PORT" cli_string="queryport=27016" />
+    <cli_param id="PLAYERS" cli_string="maxplayers=" />
+  </cli_params>
+  <cli_allow_chars>"'</cli_allow_chars>
+  <!-- escaped by default: \ " ' | &amp; ; > < ` $ ( ) [ ] -->
+  <max_user_amount>250</max_user_amount>
+  <mods>
+    <mod key="wurmu">
+      <name>none</name>
+      <installer_name>402370</installer_name>
+      <installer_login>anonymous</installer_login>
+    </mod>
+  </mods>
+  <server_params>
+    <param id="SERVERNAME" key="servername=" type="text">
+      <option>ns</option>
+      <desc>The name of the server that will be visible in server browsers. (NOTE: If the server name contains spaces, quotes are required! ie. "Server Name")</desc>
```

Comment:
you can use different options to do what you want, I think it is:

        nothing / no value = placed as is
        s = space / separated
        q = quoted
        sq = space and quotes
        sc = space and ends with a comma
        sqc = space, quoted, and ends with a comma